app/views/participants/fields/_family_history.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= f.inputs :name => "Family History Information" do -%>
 
-    <%= f.input :first_generation, :as => :select, :label => "Did parents graduate from college?", :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } %>
+	<%- person_collection_string = current_user.person_type == "Participant" ? "I am" : "Student is" -%>
+
+    <%= f.input :first_generation, :as => :select, :collection => { "No (#{person_collection_string} first generation to college)" => true, "Yes (#{person_collection_string} NOT first generation to college)" => false }%>
```

Comment:
Indenting is different on this line.

Also, we always got so many questions about this field, it might be useful for a customer to define a definition of "first generation", we'll definitely want to utilize the `HelpText` here